Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/client.py", line 541, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import yfinance as yf
import matplotlib.pyplot as plt
import matplotlib.dates as mdates
import mplfinance as mpf

vst = yf.Ticker("VST.CN")
data = vst.history(interval="1d", start="08-08-2020", end="09-08-2020")
mpf.plot(
    data,
    type="candle",
    mav=4,
    title="Victory Square Purchases Dates",
    vlines=dict(
        vlines=["08-26-2020", "09-03-2020", "09-08-2020"], linewidths=(0.5, 0.5, 0.5)
    ),
)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m<ipython-input-1-c9422a58bfcd>[0m in [0;36m<module>[0;34m[0m
[1;32m      5[0m [0;34m[0m[0m
[1;32m      6[0m [0mvst[0m [0;34m=[0m [0myf[0m[0;34m.[0m[0mTicker[0m[0;34m([0m[0;34m"VST.CN"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 7[0;31m [0mdata[0m [0;34m=[0m [0mvst[0m[0;34m.[0m[0mhistory[0m[0;34m([0m[0minterval[0m[0;34m=[0m[0;34m"1d"[0m[0;34m,[0m [0mstart[0m[0;34m=[0m[0;34m"08-08-2020"[0m[0;34m,[0m [0mend[0m[0;34m=[0m[0;34m"09-08-2020"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      8[0m mpf.plot(
[1;32m      9[0m     [0mdata[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/investing_book/lib/python3.7/site-packages/yfinance/base.py[0m in [0;36mhistory[0;34m(self, period, interval, start, end, prepost, actions, auto_adjust, back_adjust, proxy, rounding, tz, **kwargs)[0m
[1;32m    120[0m             [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    121[0m                 start = int(_time.mktime(
[0;32m--> 122[0;31m                     _time.strptime(str(start), '%Y-%m-%d')))
[0m[1;32m    123[0m             [0;32mif[0m [0mend[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    124[0m                 [0mend[0m [0;34m=[0m [0mint[0m[0;34m([0m[0m_time[0m[0;34m.[0m[0mtime[0m[0;34m([0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/investing_book/lib/python3.7/_strptime.py[0m in [0;36m_strptime_time[0;34m(data_string, format)[0m
[1;32m    569[0m     """Return a time struct based on the input string and the
[1;32m    570[0m     format string."""
[0;32m--> 571[0;31m     [0mtt[0m [0;34m=[0m [0m_strptime[0m[0;34m([0m[0mdata_string[0m[0;34m,[0m [0mformat[0m[0;34m)[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    572[0m     [0;32mreturn[0m [0mtime[0m[0;34m.[0m[0mstruct_time[0m[0;34m([0m[0mtt[0m[0;34m[[0m[0;34m:[0m[0mtime[0m[0;34m.[0m[0m_STRUCT_TM_ITEMS[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    573[0m [0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/investing_book/lib/python3.7/_strptime.py[0m in [0;36m_strptime[0;34m(data_string, format)[0m
[1;32m    357[0m     [0;32mif[0m [0;32mnot[0m [0mfound[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    358[0m         raise ValueError("time data %r does not match format %r" %
[0;32m--> 359[0;31m                          (data_string, format))
[0m[1;32m    360[0m     [0;32mif[0m [0mlen[0m[0;34m([0m[0mdata_string[0m[0;34m)[0m [0;34m!=[0m [0mfound[0m[0;34m.[0m[0mend[0m[0;34m([0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    361[0m         raise ValueError("unconverted data remains: %s" %

[0;31mValueError[0m: time data '08-08-2020' does not match format '%Y-%m-%d'
ValueError: time data '08-08-2020' does not match format '%Y-%m-%d'

